symfony/framework-bundle/3.3/src/Kernel.php

```diff
@@ -25,9 +25,19 @@ public function getLogDir()
         return $this->getProjectDir().'/var/log';
     }
 
+    private function getConfigDir()
+    {
+        return $this->getProjectDir().'/config';
+    }
+
+    private function getBundlesFile()
+    {
+        return $this->getConfigDir().'/bundles.php';
+    }
```

Comment:
Can you move these methods at the end of the class (we tend to put private methods after public and protected ones).